File: Cpp/ReadWrite_2/ReadWrite_2.cpp
#include <iostream>
#include <fstream>
#include <string>

using namespace std;

int main()
{
    // ctrl + R, ctrl + R -> change name for all
    fstream inFile("./numbers.txt", ios::in);
    fstream outFile("./results.txt", ios::out);
    string line;
    int sum = 0;

    if (!inFile.is_open()) {
        cout << "error: cannot open file" << endl;
        return 1;
    }

    while (getline(inFile, line)) {
        int newVal = stoi(line) - 100;
        outFile << newVal << endl;
    }

    //cout << "sum is " << sum << endl;

    inFile.close();
    outFile.close();
}

File: python/read_write_2.py

sum = 0
numbers = []

# file in read mode
with open("./data/numbers.txt", "r") as file:
    for line in file:
        numbers.append(line)
        
print(numbers)

# file in write mode
with open("./data/results.txt", "w") as file:
    for elem in numbers:
        newVal = int(elem) - 100
        file.write(str(newVal) + "\n")
